.dev/CRAN_Release.cmd

```diff
@@ -633,16 +633,23 @@ bunzip2 inst/tests/*.Rraw.bz2  # decompress *.Rraw again so as not to commit com
 # 9. Take sha from step 8 and run `git tag 1.16.0 96c..sha..d77` then `git push origin 1.16.0` (not `git push --tags` according to https://stackoverflow.com/a/5195913/403310)
 ######
 
+###### Branching policy for PATCH RELEASE
+# Patch releases handle regressions and CRAN-required off-cycle fixes (e.g. when changes to r-devel break data.table).
+# * Start a branch `patch-x.y.z` corresponding to the target release version. It should be branched from the previous CRAN release,
+#   e.g. branch `patch-1.15.4` from tag `v1.15.2`, `patch-1.15.6` from tag `v1.15.4`, etc.
+# * Add fixes for issues tagged to the corresponding milestone as PRs targeting the `master` branch. These PRs should be reviewed as usual
+#   and include a NEWS item under the ongoing development release.
+# * After merging a patch PR, `cherry-pick` the corresponding commit into the target patch branch. Edit the NEWS item to be under the heading
+#   for the patch release.
+
 ###### Bump dev for PATCH RELEASE
-## WARNING: review this process during the next first patch release (x.y.2) from a regular release (x.y.0), possibly during 1.15.2 release.
 # 0. Close milestone to prevent new issues being tagged with it. The final 'release checks' issue can be left open in a closed milestone.
 # 1. Check that 'git status' shows 4 files in modified and uncommitted state: DESCRIPTION, NEWS.md, init.c and this .dev/CRAN_Release.cmd
 # 2. Bump patch version in DESCRIPTION to next odd number. Note that DESCRIPTION was in edited and uncommitted state so even number never appears in git.
-# 3. Add new heading in NEWS for the next dev PATCH version. Add "(submitted to CRAN on <today>)" on the released heading.
+# 3. Add "(submitted to CRAN on <today>)" on the released heading for the NEWS.
 # 4. Bump patch version in dllVersion() in init.c
 # 5. Bump 3 patch version numbers in Makefile
-# 6. Search and replace this .dev/CRAN_Release.cmd to update 1.14.99 to 1.15.99
-# 7. Another final gd to view all diffs using meld. (I have `alias gd='git difftool &> /dev/null'` and difftool meld: http://meldmerge.org/)
-# 8. Push to master with this consistent commit message: "1.15.0 on CRAN. Bump to 1.15.99"
-# 9. Take sha from step 8 and run `git tag 1.14.8 96c..sha..d77` then `git push origin 1.14.8` (not `git push --tags` according to https://stackoverflow.com/a/5195913/403310)
+# 6. Another final gd to view all diffs using meld. (I have `alias gd='git difftool &> /dev/null'` and difftool meld: http://meldmerge.org/)
+# 7. Push to master with this consistent commit message: "1.15.{2,4,6,...} on CRAN. Bump to 1.15.99"
```

Comment:
This might be meticulous but shouldn't it be "merge to master", since master might contain other fixed bugs/new features etc.

Reply:
Hmm actually I think we should just remove this step. We no longer push/merge these to master. patch branches continue out from each other, with individual commits mirrored from `master`. cc @TysonStanley to confirm.

Reply:
Yes, I agree @MichaelChirico . No PR associated with the patch to master. Should we note, however, which commits are relevant for master in an issue?

Reply:
Would that be covered by the issues/PRs tagged with the right milestone? I might not be fully catching your meaning

Reply:
Oh yep, ignore that. Still getting used the milestone tool on here. Should be good to go.

Reply:
Which commit gets pushed as the patch branch tag? are the patch branch commits squashed? do we need to keep the patch branch around or can it be deleted?

Reply:
It's usually the last commit of the CRAN release, with version changes in init.c or it's the NEWS file date. Either one seems like a good "final" commit of the patch. Thoughts?

Reply:
Also think after we tag it, we don't need the branch anymore?

Reply:
SGTM, updated the steps